cache.go

```diff
@@ -85,7 +85,7 @@ func (c *Cache) Get(key string) *Item {
 	if item == nil {
 		return nil
 	}
-	if item.expires > time.Now().UnixNano() {
+	if item.Expires().After(time.Now()) {
```

Comment:
I believe this is equivalent but thread safe, please double check.

Reply:
Can use:
```
if !item.Expired() {
 ...
```

Reply:
I like that more. Let me update it.

Reply:
Done.